integration/feature/invalidation-foreign/test/src/ScriptsInvalidationForeignTests.scala

```diff
@@ -1,66 +0,0 @@
-package mill.integration
```

Comment:
The scenario that this test covers doesn't really apply anymore, given the new desugaring of the `build.sc` files